src/GoTrueApi.ts

```diff
@@ -203,6 +203,34 @@ export default class GoTrueApi {
     }
   }
 
+  /**
+   * Logs in an OpenID Connect user using their id_token.
+   * @param id_token The id_token of the user.
+   * @param nonce The nonce of the user.
```

Comment:
Let's make it clear.

The nonce is not automatically generated by the initial grant as a function of supabase, but is a random value generated by the developer himself "before" the initial grant is started.

There is no way that an app developer can be unaware of a nonce that he or she has generated on the fly.

However, it is expected that developers who do not understand the OpenID Connect specification will ask what a nonce is. To deal with this, it might be a good idea to specify the following points in the supabase change log or documentation.

1) The nonce is a value randomly generated by the developer before the initial grant is started.
2) You should check the OpenID Connect specification for details. https://openid.net/developers/specs/

I personally think that developers who do not understand what nonce is should not use this method.